notifications/sinks.go

```diff
@@ -147,6 +184,9 @@ func (eq *eventQueue) Write(events ...Event) error {
 	}
 	eq.events.PushBack(events)
 	eq.cond.Signal() // signal waiters
+	n := float64(len(events))
```

Comment:
I assume every message the broadcaster handles ends up in the eventQueue at some point, correct?
If not, we probably should instrument the broadcaster as well.